feat: added invisible type

Grafikart opened the pull request:
- Add a new type to handle invisible recaptcha v2 (Fix #23).
- Constraint is added to the type by default (why you wouldn't want that ?)

Grafikart commented:
I chose to inject JavaScript "on the fly" for convenience but i'm not sure how I feel about this.

garak commented:
I'm sure you can find a way to avoid javascript inside twig

Grafikart commented:
@garak I removed JavaScript and I added instructions for invisible Recaptcha

bernard-ng commented:
hello @garak can you merge @Grafikart , this feature is very interesting

garak commented:
> hello @garak can you merge @Grafikart , this feature is very interesting

Currently I can't.
There's a review that needs to be addressed, and a conflict that needs to be solved

Grafikart commented:
Since I'm stupid I deleted my fork and I can no longer continue this  pull request. I opened a new one with the same edits, and fixed my code according to your review) #37
